install/playbooks/roles/ldap/tasks/main.yml

```diff
@@ -100,6 +100,7 @@
   vars:
     cert_dir: 'ldap.{{ network.domain }}'
     entity_group: openldap
+    access_private_key: true
```

Comment:
I am planning to set the default to false. Otherwise, I noticed the CI failed before because the certificate file permissions were changed from the correct ones to the wrong ones by one of the next stage. I did not identified the guilty stage, but this was more a reinsurance, and a step in the direction to set the default to false.

Reply:
Ok, I understand. I agree with the goal of defaulting to false.
It's on my list to test which servers really require to have their user/group accessing their private key, because some (most) of them load it when they start and bind as root.